src/Pos/Wallet/Web/Server/Methods.hs

```diff
@@ -813,7 +813,7 @@ selectSrcAddresses allAddrs outputCoins (TxFee fee) =
         | reqCoins == mkCoin 0 =
             Left "Spending non-positive amount of money!"
         | [] <- addresses =
-            Left $ sformat ("Not enough money (need " %build % " more)") reqCoins
+            Left "It's not allowed to send money to the same address you are sending from. Make sure you have enough addresses with money in this account or send to a different address."
```

Comment:
Nope, that won't do. This error is also displayed when user _actually_ doesn't have enough money - when trying to send too much to _another_ wallet. Here's the testing process:

```
> var api = require('../output/Daedalus.ClientApi')
undefined
> var tls = api.tlsInit('../ca.crt')
undefined
> api.importWallet(tls, '/home/flyingleafe/Job/cardano-sl/secrets/secret-gen-2.key', null).then(console.log).catch(console.log)
Promise { <pending> }
> { cwPassphraseLU: 1500487824.3412707,
  cwMeta: { cwUnit: 0, cwName: 'Genesis wallet', cwAssurance: 'CWANormal' },
  cwId: '1fhDeyXxm62bLVnhsBYP4Dscwh5hF9r2R3iRnwKKV5fCk3W',
  cwHasPassphrase: false,
  cwAmount: { getCCoin: '100' },
  cwAccountsNumber: 1 }

> api.restoreWallet(tls, 'test', 'CWANormal', 0, 'reform claw bubble quantum fossil economy example bottom journey claw nurse three', null).then(console.log).catch(console.log)
Promise { <pending> }
> { cwPassphraseLU: 1500487844.5445518,
  cwMeta: { cwUnit: 0, cwName: 'test', cwAssurance: 'CWANormal' },
  cwId: '1g71WWc9wpqJomgh6Dc2HEoiEPrJ9gQKkp3ZVgxRhPpD9Tp',
  cwHasPassphrase: false,
  cwAmount: { getCCoin: '0' },
  cwAccountsNumber: 1 }

> api.getAccounts(tls).then(console.log).catch(console.log)
Promise { <pending> }
> [ { caMeta: { caName: 'Genesis account' },
    caId: '1fhDeyXxm62bLVnhsBYP4Dscwh5hF9r2R3iRnwKKV5fCk3W@0',
    caAmount: { getCCoin: '100' },
    caAddresses: [ [Object] ] },
  { caMeta: { caName: 'Initial account' },
    caId: '1g71WWc9wpqJomgh6Dc2HEoiEPrJ9gQKkp3ZVgxRhPpD9Tp@0',
    caAmount: { getCCoin: '0' },
    caAddresses: [ [Object] ] } ]

> api.getAccount(tls, '1g71WWc9wpqJomgh6Dc2HEoiEPrJ9gQKkp3ZVgxRhPpD9Tp@0').then(console.log)
Promise { <pending> }
> { caMeta: { caName: 'Initial account' },
  caId: '1g71WWc9wpqJomgh6Dc2HEoiEPrJ9gQKkp3ZVgxRhPpD9Tp@0',
  caAmount: { getCCoin: '0' },
  caAddresses: 
   [ { cadIsUsed: false,
       cadIsChange: false,
       cadId: '1mCDtHCNfnzk2xg5SLKJqqcXZYsYTmzMgmmksp247mjwm7op9hwZBbAbvejhF12UTpwRufV6drN7mUWVo6jA',
       cadAmount: [Object] } ] }

> api.newPayment(tls, '1fhDeyXxm62bLVnhsBYP4Dscwh5hF9r2R3iRnwKKV5fCk3W@0', '1mCDtHCNfnzk2xg5SLKJqqcXZYsYTmzMgmmksp247mjwm7op9hwZBbAbvejhF12UTpwRufV6drN7mUWVo6jA', 200, null).then(console.log).catch(console.log)
Promise { <pending> }
> Error: ServerError: Pos.Wallet.Web.Error.Types.RequestError "It's not allowed to send money to the same address you are sending from. Make sure you have enough addresses with money in this account or send to a different address."
    at Object.exports.error (/home/flyingleafe/Job/cardano-sl/daedalus/output/Control.Monad.Eff.Exception/foreign.js:8:10)
    at mkServerError (/home/flyingleafe/Job/cardano-sl/daedalus/output/Daedalus.BackendApi/index.js:90:44)
    at /home/flyingleafe/Job/cardano-sl/daedalus/output/Data.Either/index.js:256:33
    at /home/flyingleafe/Job/cardano-sl/daedalus/output/Data.Either/index.js:230:24
    at /home/flyingleafe/Job/cardano-sl/daedalus/output/Daedalus.BackendApi/index.js:98:76
    at /home/flyingleafe/Job/cardano-sl/daedalus/output/Data.Either/index.js:358:16
    at /home/flyingleafe/Job/cardano-sl/daedalus/output/Data.Function/index.js:16:24
    at /home/flyingleafe/Job/cardano-sl/daedalus/output/Control.Bind/index.js:42:48
    at /home/flyingleafe/Job/cardano-sl/daedalus/output/Daedalus.BackendApi/index.js:116:211
    at /home/flyingleafe/Job/cardano-sl/daedalus/output/Control.Monad.Aff/foreign.js:182:21
```

So if we need to distinguish between these two types of error, we need to do it somewhere else.